### ISSUE TITLE
A11y: Non-interactive element <xx> cannot have interactive role 'xxx'

### ISSUE THREAD

**Issue by hslee2008** [carbon-design-system/carbon-components-svelte#1704]
Am I suppose to change the configuration to ignore these errors or should there be changes made to carbon-components-svelte?

Or should it be ignored something like in this [a11y commit](https://github.com/carbon-design-system/carbon-components-svelte/commit/c6373dc9b6db53a73784e8e480d8f6e2bc278948)

https://github.com/evcohen/eslint-plugin-jsx-a11y/blob/main/docs/rules/no-noninteractive-element-to-interactive-role.md

```svelte
transforming (101) node_modules\carbon-components-svelte\src\Button\ButtonSkeleton.svelte9:19:38 PM [vite-plugin-svelte] /node_modules/carbon-components-svelte/src/ContextMenu/ContextMenu.svelte:156:0 A11y: Non-interactive element <ul> cannot have interactive r
ole 'menu'
154: />
155:
156: <ul
     ^
157:   bind:this="{ref}"
158:   role="menu"
transforming (174) node_modules\carbon-components-svelte\src\ListBox\ListBoxMenuItem.svelte9:19:41 PM [vite-plugin-svelte] /node_modules/carbon-components-svelte/src/OverflowMenu/OverflowMenu.svelte:245:4 A11y: Non-interactive element <ul> cannot have interacti
ve role 'menu'
243:   </slot>
244:   {#if open}
245:     <ul
         ^
246:       bind:this="{menuRef}"
247:       role="menu"
transforming (181) node_modules\carbon-components-svelte\src\Popover\Popover.svelte9:19:41 PM [vite-plugin-svelte] /node_modules/carbon-components-svelte/src/ProgressIndicator/ProgressStep.svelte:56:2 A11y: The attribute 'aria-disabled' is not supported by the
role 'listitem'. This role is implicit on the element <li>.
54: <!-- svelte-ignore a11y-mouse-events-have-key-events -->
55: <li
56:   aria-disabled="{disabled}"
      ^
57:   id="{id}"
58:   class:bx--progress-step="{true}"
transforming (198) node_modules\carbon-components-svelte\src\SkeletonText\SkeletonText.svelte9:19:42 PM [vite-plugin-svelte] /node_modules/carbon-components-svelte/src/Tabs/Tabs.svelte:154:2 A11y: Non-interactive element <ul> cannot have interactive role 'tabli
st'
152:     <ChevronDown aria-hidden="true" title="{iconDescription}" />
153:   </div>
154:   <ul
       ^
155:     bind:this="{refTabList}"
156:     role="tablist"
transforming (229) node_modules\carbon-components-svelte\src\Tooltip\Tooltip.svelte9:19:43 PM [vite-plugin-svelte] /node_modules/carbon-components-svelte/src/TreeView/TreeView.svelte:191:0 A11y: Non-interactive element <ul> cannot have interactive role 'tree'
189: {/if}
190:
191: <ul
     ^
192:   {...$$restProps}
193:   role="tree"
transforming (238) node_modules\carbon-components-svelte\src\UIShell\Header.svelte9:19:43 PM [vite-plugin-svelte] /node_modules/carbon-components-svelte/src/UIShell/HeaderNav.svelte:9:2 A11y: Non-interactive element <ul> cannot have interactive role 'menubar'
 7:
 8: <nav {...props} class:bx--header__nav="{true}" {...$$restProps}>
 9:   <ul {...props} role="menubar" class:bx--header__menu-bar="{true}">
      ^
10:     <slot />
11:   </ul>
9:19:43 PM [vite-plugin-svelte] /node_modules/carbon-components-svelte/src/UIShell/HeaderNavMenu.svelte:97:2 A11y: Non-interactive element <ul> cannot have interactive role 'menu'
 95:     <ChevronDown class="bx--header__menu-arrow" />
 96:   </a>
 97:   <ul
       ^
 98:     bind:this="{menuRef}"
 99:     role="menu"
transforming (248) node_modules\carbon-components-svelte\src\UIShell\SideNav.svelte9:19:43 PM [vite-plugin-svelte] /node_modules/carbon-components-svelte/src/UIShell/SideNavMenu.svelte:51:2 A11y: Non-interactive element <ul> cannot have interactive role 'menu'
49:     </div>
50:   </button>
51:   <ul
      ^
52:     role="menu"
53:     class:bx--side-nav__menu="{true}"
9:19:44 PM [vite-plugin-svelte] /node_modules/carbon-components-svelte/src/UIShell/HeaderSearch.svelte:137:4 A11y: Non-interactive element <ul> cannot have interactive role 'menu'
135:
136:   {#if active && results.length > 0}
137:     <ul aria-labelledby="search-label" role="menu" id="search-menu">
         ^
138:       {#each results as result, i}
139:         <li role="none">
transforming (290) node_modules\carbon-components-svelte\src\icons\Warning.svelte9:19:44 PM [vite-plugin-svelte] /node_modules/carbon-components-svelte/src/TreeView/TreeViewNodeList.svelte:75:2 A11y: Non-interactive element <li> cannot have interactive role 'tr
eeitem'
73:   {/each}
74: {:else}
75:   <li
      ^
76:     bind:this="{ref}"
77:     role="treeitem"
transforming (302) src\css\custom.css9:19:44 PM [vite-plugin-svelte] /node_modules/carbon-components-svelte/src/TreeView/TreeViewNode.svelte:78:0 A11y: Non-interactive element <li> cannot have interactive role 'treeitem'
76: </script>
77:
78: <li
    ^
79:   bind:this="{ref}"
80:   role="treeitem"
```

**Comment by theetrain** [comment]
Resolved in [v0.73.5](https://github.com/carbon-design-system/carbon-components-svelte/releases/tag/v0.73.5).
